File: domain/todo.go
package domain

import (
	"time"
)

type Todo struct {
	Id          string    `json:"trans_id"`
	UserId      string    `json:"user_id"`
	Description string    `json:"description"`
	Status      string    `json:"status"`
	CreatedAt   time.Time `json:"created_at"`
	UpdatedAt   time.Time `json:"updated_at"`
}

File: ToDo/Domain/DBModels/Todo.cs
using ToDo.Api.Domain.Core;

namespace ToDo.Api.Domain.DBModels
{
    public class Todo : Entity
    {
        public Guid UserId { get; set; }
        public int Status { get; set; }
        public string? TodoName { get; set; }
        public string? TodoDescription { get; set; }
        public DateTime? DateCreated { get; set; }
    }
}
